### ISSUE TITLE
Mamba installing issues on GH actions for Ubuntu 20.04 and 22.04, since Conda version 22.11.1

### ISSUE THREAD

**Issue by bogdan-damian-bgd** [conda/conda#12303]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

Since the recent (mini)Conda version 22.11.1 in the GitHub Actions runners for [Ubuntu 20.04](https://github.com/actions/runner-images/blob/4a42f07ad2b808b15da8222136632b434cefe328/images/linux/Ubuntu2004-Readme.md) and [22.04](https://github.com/actions/runner-images/blob/072cca75ce151e04ee69f0ab0cabac049dde4b61/images/linux/Ubuntu2204-Readme.md) images, installing Mamba (using `conda install --yes -c conda-forge mamba`) doesn't work (see workflow fail examples for [20.04](https://github.com/bogdan-damian-bgd/action-tests/actions/runs/4176481769/jobs/7232824374) and [22.04](https://github.com/bogdan-damian-bgd/action-tests/actions/runs/4176483922/jobs/7232828956)).

Downgrading Conda to previous version 4.12.0 seems like workaround for now, so the Mamba installation passes (see workflows [d20.04](https://github.com/bogdan-damian-bgd/action-tests/actions/runs/4176495054/jobs/7232854236), [d22.04](https://github.com/bogdan-damian-bgd/action-tests/actions/runs/4176495851/jobs/7232855873)) - hint found in issue #9367 

Installing Mamba on the latest Conda version 23.1.0 also seems to fail ([n20.04](https://github.com/bogdan-damian-bgd/action-tests/actions/runs/4176497715/jobs/7232860195)).

Related GH Actions issue: https://github.com/actions/runner-images/issues/6910

### Conda Info

```shell
active environment : None
       user config file : /home/runner/.condarc
 populated config files : 
          conda version : 22.11.1
    conda-build version : not installed
         python version : 3.10.8.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.31=0
                          __linux=5.15.0=0
                          __unix=0=0
       base environment : /usr/share/miniconda  (writable)
      conda av data dir : /usr/share/miniconda/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /usr/share/miniconda/pkgs
                          /home/runner/.conda/pkgs
       envs directories : /usr/share/miniconda/envs
                          /home/runner/.conda/envs
               platform : linux-64
             user-agent : conda/22.11.1 requests/2.28.1 CPython/3.10.8 Linux/5.15.0-1031-azure ubuntu/20.04.5 glibc/2.31
                UID:GID : 1001:122
             netrc file : None
           offline mode : False
```

### Conda list

```shell
# packages in environment at /usr/share/miniconda:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
_openmp_mutex             5.1                       1_gnu    defaults
brotlipy                  0.7.0           py310h7f8727e_1002    defaults
bzip2                     1.0.8                h7b6447c_0    defaults
ca-certificates           2022.10.11           h06a4308_0    defaults
certifi                   2022.12.7       py310h06a4308_0    defaults
cffi                      1.15.1          py310h5eee18b_3    defaults
charset-normalizer        2.0.4              pyhd3eb1b0_0    defaults
conda                     22.11.1         py310h06a4308_4    defaults
conda-content-trust       0.1.3           py310h06a4308_0    defaults
conda-package-handling    1.9.0           py310h5eee18b_1    defaults
cryptography              38.0.1          py310h9ce1e76_0    defaults
idna                      3.4             py310h06a4308_0    defaults
ld_impl_linux-64          2.38                 h1181459_1    defaults
libffi                    3.4.2                h6a678d5_6    defaults
libgcc-ng                 11.2.0               h1234567_1    defaults
libgomp                   11.2.0               h1234567_1    defaults
libstdcxx-ng              11.2.0               h1234567_1    defaults
libuuid                   1.41.5               h5eee18b_0    defaults
ncurses                   6.3                  h5eee18b_3    defaults
openssl                   1.1.1s               h7f8727e_0    defaults
pip                       22.3.1          py310h06a4308_0    defaults
pluggy                    1.0.0           py310h06a4308_1    defaults
pycosat                   0.6.4           py310h5eee18b_0    defaults
pycparser                 2.21               pyhd3eb1b0_0    defaults
pyopenssl                 22.0.0             pyhd3eb1b0_0    defaults
pysocks                   1.7.1           py310h06a4308_0    defaults
python                    3.10.8               h7a1cb2a_1    defaults
readline                  8.2                  h5eee18b_0    defaults
requests                  2.28.1          py310h06a4308_0    defaults
ruamel.yaml               0.17.21         py310h5eee18b_0    defaults
ruamel.yaml.clib          0.2.6           py310h5eee18b_1    defaults
setuptools                65.5.0          py310h06a4308_0    defaults
six                       1.16.0             pyhd3eb1b0_1    defaults
sqlite                    3.40.0               h5082296_0    defaults
tk                        8.6.12               h1ccaba5_0    defaults
toolz                     0.12.0          py310h06a4308_0    defaults
tqdm                      4.64.1          py310h06a4308_0    defaults
tzdata                    2022g                h04d1e81_0    defaults
urllib3                   1.26.13         py310h06a4308_0    defaults
wheel                     0.37.1             pyhd3eb1b0_0    defaults
xz                        5.2.8                h5eee18b_0    defaults
zlib                      1.2.13               h5eee18b_0    defaults
```

**Comment by travishathaway** [comment]
Hi @bogdan-damian-bgd,

Thank you for submitting this bug report. I would really like to see the error messages from the failed runners, but the links you shared are no longer valid. Would it be possible for you to paste the relevant parts of the error here?

**Comment by bogdan-damian-bgd** [comment]
Hi @travishathaway, I updated the links in the issue description, where you can find workflows and logs for Ubuntu20 and 22.
Also, the failed installation log with Ubuntu20 can be found in [this file](https://github.com/bogdan-damian-bgd/action-tests/blob/main/logs/Install%20Mamba%20on%20new%20Conda%20(22.11.1).log) (when using [this](https://github.com/bogdan-damian-bgd/action-tests/blob/main/.github/workflows/test-mamba-new-01a.yml) GH Actions workflow)

**Comment by travishathaway** [comment]
Hi @bogdan-damian-bgd,

Thanks for giving me an update on this issue. I had to take some time to replicate this myself, and I did this on a personal repository. It looks like the error is primarily a dependency resolution issue. Unfortunately, I am unable to replicate it.

This is the pull request that I used to test out the Ubuntu based test runner:

- https://github.com/travishathaway/latz/pull/2

Here's some pointers I have for you to get this running better in the future:

- Be sure to always update conda to the latest version with `conda update -n base -c defaults conda`
- Sometimes it helps to install additional dependencies without the `-c` flag and instead specify it like `conda-forge::mamba`. In this form, conda-forge with only be used to retrieve the necessary package (i.e. `mamba`)

I will be closing this issue now as it does not represent an actual bug within conda itself that can be fixed. It instead represents a problem with how the particular runtime environment is configured and is therefore considered https://github.com/conda/conda/labels/off-topic for this repository.

I hope these pieces of advice help. If you truly believe this to be a bug within the conda package manager itself, I invite you to re-open this issue but also ask that you provide more evidence to support your claim.
